sno/output_util.py

```diff
@@ -69,3 +89,71 @@ def is_empty_stream(stream):
             return True
         stream.seek(pos)
     return False
+
+
+def _setenv(k, v):
+    if v is None:
+        del os.environ[k]
+    else:
+        os.environ[k] = v
+
+
+@contextmanager
+def _push_environment(k, v):
+    orig = os.environ.get(k)
+    _setenv(k, v)
+    try:
+        yield
+    finally:
+        _setenv(k, orig)
+
+
+DEFAULT_PAGER = shutil.which('less')
```

Comment:
Ah, click doesn't have a pager, it passes it to `less`/etc?

Reply:
correct. but it does handle the Weird Issues around doing that